webpack.config.js

```diff
@@ -76,7 +76,13 @@ const BROWSER_CONFIG = Object.assign({}, BASE_CONFIG, {
     fs: 'empty'
   },
 
-  plugins: []
+  plugins: [
+    new webpack.DefinePlugin({
+      // Overwrite the default env var so that we can test access token handling
+      'process.env.MapboxAccessToken': JSON.stringify(''),
+      'process.env.MAPBOX_ACCESS_TOKEN': JSON.stringify(process.env.MapboxAccessToken)
```

Comment:
I thought we decided to drop support for the all CAPS version some versions ago. Am I mistaken, or are we adding it back?

Reply:
This is for testing only. If `MapboxAccessToken` is set then the map will automatically fallback to using it, and we have no way to test the no-token scenario.